charts/testmachinery/charts/gardener-resource-manager/values.yaml

```diff
@@ -0,0 +1,22 @@
+images:
+  gardener-resource-manager: eu.gcr.io/gardener-project/gardener/gardener-resource-manager:v0.12.0
+
+resources:
+  requests:
+    cpu: 23m
+    memory: 47Mi
```

Comment:
That looks like magic numbers, but...OK :)

Reply:
just copied values. but I gues its ok to make them not look htat magically:)